README.md

```diff
@@ -20,8 +20,14 @@ If you'd like to help out, please see [CONTRIBUTING.md](CONTRIBUTING.md).
 
 ## Release history
 
-* Next release:
+* Release 0.21.2 (2023-06-14)
   - Bump MSRV to 1.60 to track similar change in dependencies.
+  - Differentiate between unexpected and expected EOF in `Stream` and `OwnedStream`.
+  - `RootCertStore::add_parsable_certificates` now takes a `&[impl AsRef<[u8]>]`.
+  - Add QUIC V2 support.
+  - Upgrade `env_logger` to v0.10.
```

Comment:
IIRC env_logger is a dev-dep only, so probably doesn't need to go in the changelog.

Reply:
Ah, right, my bad.